Extension/src/LanguageServer/configurations.ts

```diff
@@ -288,7 +291,8 @@ export class CppProperties {
     }
 
     private applyDefaultIncludePathsAndFrameworks(): void {
-        if (this.configurationIncomplete && this.defaultIncludes && this.defaultFrameworks && this.vcpkgPathReady) {
+        if (this.configurationIncomplete && this.defaultIncludes && this.defaultFrameworks && this.vcpkgPathReady
+            && this.nodeAddonIncludePathsReady) {
```

Comment:
This variable is no longer needed. It was only needed when the applyDefaultConfigurationValues was being used.

Reply:
yes, obvious when i see it. you have a much better handle on all the moving pieces than i do - thanks for spending time on this.